developer_manual/digging_deeper/dashboard.rst

```diff
@@ -0,0 +1,178 @@
+=========
+Dashboard
+=========
+
+The dashboard app aims to provide the user with a general overview of their
```

Comment:
nit: wanna link to the app on the app store for those who want to have a quick look at what that is?

Reply:
It will be shipped, I'll extend with screenshots once we have something nice ;)